server/routes/induction/create/inPrisonTrainingCreateController.test.ts

```diff
@@ -97,47 +93,6 @@ describe('inPrisonTrainingCreateController', () => {
       expect(req.session.inPrisonTrainingForm).toBeUndefined()
       expect(req.session.inductionDto).toEqual(inductionDto)
     })
-
-    it('should get the In Prison Training view given the previous page was Check Your Answers', async () => {
```

Comment:
We can no longer do this type of test because the controller no longer manages state of what the previous page was